### PR TITLE
Fix footer display

### LINKED ISSUE #24249: Footer dispaly issue
### Description

image

It seems that footer has been changed. (In the middle of the footer, not left and right)

Then if we adjust the browser's width:
image
It seems that there's an issue.

### Gitea Version

1.20.0+dev-517-g948a9ee5e

### Can you reproduce the bug on the Gitea demo site?

No

### Log Gist

_No response_

### Screenshots

_No response_

### Git Version

_No response_

### Operating System

_No response_

### How are you running Gitea?

build

### Database

None

### PR DESCRIPTION
**Pull request by wxiaoguang**
Fix #24249

Diff with ignoring spaces: https://github.com/go-gitea/gitea/pull/24251/files?diff=split&w=1

Screenshots:

<details>

image

image

</details>

### CODE REVIEW

**Comment by silverwind** [comment]
Regression from https://github.com/go-gitea/gitea/pull/24184?

**Comment by silverwind** [comment]
Ok than backport is needed. Strange that I did not see it in my testing.

File: web_src/css/home.css [hunk 1]
```diff
@@ -43,9 +43,38 @@
   color: var(--color-green);
 }
 
+
```
Comment (silverwind) [inline web_src/css/home.css]:
```suggestion
```

**Review by silverwind — APPROVED** [review]
Didn't see a rendering difference in the footer layout, but changes still look good.

**Review by lunny — APPROVED** [review]

**Comment by silverwind** [comment]
.

**Pull request merged**